test/lib/gatewaylib/bytes32_to_bytes.js

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+const GatewayLib = artifacts.require('GatewayLib');
```

Comment:
In `GatewayLib.sol`
1. Why is `bytes32ToBytes` public function and not external?
2. Fix the documentation of `bytes32ToBytes` as per the style guide.
3. Use return variable in `bytes32ToBytes`.